assessment/services/test_flow.py

```diff
@@ -20,18 +21,18 @@ def validate_test_flow_registration(request_data: dict):
             tool_release_time = tool_used_data['release_time']
 
             try:
-                utils.get_tool_from_id(tool_id)
+                utils.get_tool_of_company_from_id(tool_id, company)
                 utils.get_time_from_date_time_string(tool_release_time)
             except (ValueError, AssessmentToolDoesNotExist) as exception:
-                raise InvalidRequestException(str(exception))
+                raise InvalidTestFlowRegistration(str(exception))
 
 
-def convert_assessment_tool_id_to_assessment_tool(request_data) -> list:
+def convert_assessment_tool_id_to_assessment_tool(request_data: dict, company: Company) -> list:
     assessment_tools_in_request = request_data.get('tools_used')
     assessment_tools_release_time = []
     if assessment_tools_in_request:
         for request_tool_data in assessment_tools_in_request:
-            tool = utils.get_tool_from_id(request_tool_data.get('tool_id'))
+            tool = utils.get_tool_of_company_from_id(request_tool_data.get('tool_id'), company)
             release_time = utils.get_time_from_date_time_string(request_tool_data.get('release_time'))
             tool_data = {
                 'tool': tool,
```

Comment:
Hello, Jo. If I may ask, does the release time means the time the assessees are notified about the assignment? I guess if that's the case I believe adding a new `start_time` might help, which signifies what time the attempt will be opened to the assessees.

Reply:
Hello, Alex. I have accomodated that in my latest commit. Thanks for your suggestion.

Reply:
Hello, Jo. Thanks for applying this change. I have also created the corresponding component to apply this in the frontend.